backend/clickhouse/logs.go

```diff
@@ -69,6 +69,8 @@ func (client *Client) BatchWriteLogRows(ctx context.Context, logRows []*LogRow)
 		if len(logRow.UUID) == 0 {
 			logRow.UUID = uuid.New().String()
 		}
+		// TODO (et) - move this logic to a builder function (#4464)
+		logRow.SeverityText = strings.ToLower(logRow.SeverityText)
```

Comment:
This partially fixes #4464. The true fix though should also limit the number of levels we support:

```sql
SELECT distinct(SeverityText) from logs;



log
--
warn
debug
ERROR
WARN
INFO
info
group
assert
timeEnd
time
trace
 
groupCollapsed
stdout
stderr
dir
error
edge-function-invocation
fatal
unknown
table
Info
groupEnd
middleware-invocation
```

We need to default everything that's not in `LogLevel` to be just `info`